.travis.yml

```diff
@@ -14,12 +15,32 @@ script: $WORKSPACE/scripts/linux/build.sh && $WORKSPACE/scripts/linux/test.sh
 matrix:
   include:
   - compiler: clang
+    name: Linux Build using clang & tests
```

Comment:
It is enough to run tests only once.
Please remove tests from clang build and only leave in gcc.

Reply:
Done. Test removed from clang